app/cafe.py

```diff
@@ -0,0 +1,26 @@
+import datetime
+
+from app.errors import (NotVaccinatedError,
+                        NotWearingMaskError,
+                        OutdatedVaccineError)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        vaccine = visitor.get("vaccine")
+        mask = visitor.get("wearing_a_mask")
+        today = datetime.date.today()
+
+        if not vaccine:
+            raise NotVaccinatedError("All friends should be vaccinated")
```

Comment:
I guess it should say that `visitor` is not vaccinated, not all friends should have vaccine